### PR TITLE
[ACM-2.11] ACM#10464: Extracting OpenShift release image digest

### PR DESCRIPTION
**Pull request by xenolinux**
https://issues.redhat.com/browse/ACM-10464

### CODE REVIEW

File: clusters/hosted_control_planes/create_cluster_bm.adoc [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
 
 +
 ----
-oc create ns <hosted-cluster-namespace>-<hosted-cluster-name>
+oc create ns <hosted_cluster_namespace>-<hosted_cluster_name>
 ----
 
 +
-Replace `<hosted-cluster-namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted-cluster-name>` with your hosted cluster name.
+Replace `<hosted_cluster_namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted-cluster-name>` with your hosted cluster name.
```
Comment (CrystalChun) [inline clusters/hosted_control_planes/create_cluster_bm.adoc]:
```suggestion
Replace `<hosted_cluster_namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted_cluster_name>` with your hosted cluster name.
```
I'd guess you'd want this variable with underscores too?

Comment (xenolinux) [reply clusters/hosted_control_planes/create_cluster_bm.adoc]: Yes correct

**Review by CrystalChun — APPROVED** [review]

File: clusters/hosted_control_planes/create_cluster_bm.adoc [hunk 2]
```diff
@@ -1,44 +1,44 @@
 [#creating-a-hosted-cluster-bm]
 = Creating a hosted cluster on bare metal
 
-You can create a hosted cluster or import one. For instructions to import a hosted cluster, see _Importing a hosted cluster_.
+You can create a hosted cluster on bare metal or import one. For instructions to import a hosted cluster, see _Importing a hosted cluster_.
 
 . Create the hosted control plane namespace by entering the following command:
 
 +
 ----
```
Comment (dockerymick) [inline clusters/hosted_control_planes/create_cluster_bm.adoc:10]:
```suggestion
[source,bash]
----
```

File: clusters/hosted_control_planes/create_cluster_bm.adoc [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
 
 +
-Replace `<hosted-cluster-namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted-cluster-name>` with your hosted cluster name.
+Replace `<hosted_cluster_namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted_cluster_name>` with your hosted cluster name.
 
 . Verify that you have a default storage class configured for your cluster. Otherwise, you might see pending PVCs. See the following example:
 
 +
 ----
 hcp create cluster agent \
```
Comment (dockerymick) [inline clusters/hosted_control_planes/create_cluster_bm.adoc:22]: https://github.com/stolostron/rhacm-docs/pull/6394#discussion_r1591192196

File: clusters/hosted_control_planes/disconnected_intro.adoc [hunk 4]
```diff
@@ -15,9 +15,9 @@ The following diagram illustrates an example architecture of a disconnected envi
 image:../images/489_RHACM_HyperShift_on_bare_metal_1223.png[Disconnected architecture diagram]
 
 . You configure infrastructure services, including the registry certificate deployment with TLS support, web server, and DNS, to ensure that the disconnected deployment works.
```
Comment (dockerymick) [inline clusters/hosted_control_planes/disconnected_intro.adoc]:
```suggestion
. Configure infrastructure services, including the registry certificate deployment with TLS support, web server, and DNS, to ensure that the disconnected deployment works.
```

File: clusters/hosted_control_planes/disconnected_intro.adoc [hunk 5]
```diff
@@ -15,9 +15,9 @@ The following diagram illustrates an example architecture of a disconnected envi
...
-. You create a config map in the `openshift-config` namespace. In this example, the config map is named `registry-config`. The content of the config map is the Registry CA certificate. The data field of the config map must contain the following key/value: 
+. You create a config map in the `openshift-config` namespace. In this example, the config map is named `registry-config`. The content of the config map is the Registry CA certificate. The data field of the config map must contain the following key/value:
```
Comment (dockerymick) [inline clusters/hosted_control_planes/disconnected_intro.adoc]:
```suggestion
. Create a config map in the `openshift-config` namespace. In this example, the config map is named `registry-config`. The content of the config map is the Registry CA certificate. The data field of the config map must contain the following key/value:
```

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: clusters/hosted_control_planes/create_cluster_bm.adoc (hunk 3 above)
Comment (xenolinux) [reply clusters/hosted_control_planes/create_cluster_bm.adoc:22]: This is a continuous one command. Added [source,bash] for this command too.

**Review by dockerymick — APPROVED** [review]
Thanks for the updates. This looks good to me!

File: clusters/hosted_control_planes/create_cluster_bm.adoc [hunk 6]
```diff
@@ -1,49 +1,52 @@
...
 ----
-oc create ns <hosted-cluster-namespace>-<hosted-cluster-name>
+oc create ns <hosted_cluster_namespace>-<hosted_cluster_name>
 ----
 
 +
-Replace `<hosted-cluster-namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted-cluster-name>` with your hosted cluster name.
+Replace `<hosted_cluster_namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted_cluster_name>` with your hosted cluster name.
 
 . Verify that you have a default storage class configured for your cluster. Otherwise, you might see pending PVCs. See the following example:
```
Comment (dockerymick) [inline clusters/hosted_control_planes/create_cluster_bm.adoc]: Can you change this to read, "Run the following command."

Comment (xenolinux) [reply clusters/hosted_control_planes/create_cluster_bm.adoc]: Fixed

**Review by dockerymick — APPROVED** [review]

File: clusters/hosted_control_planes/create_cluster_bm.adoc (hunk 6 above)
Comment (xenolinux) [reply clusters/hosted_control_planes/create_cluster_bm.adoc]: Updated PRs with this change for all versions - 2.9, 2.10, and 2.11

**Pull request merged**